.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -46,8 +45,8 @@ body:
       description: How did you install Bottles?
       options:
         - Flatpak from Flathub
-        - From source
-        - Other, unofficial package
+        - Flatpak from Flathub Beta
+        - Flatpak from GitHub Artifacts
```

Comment:
Debatable, not really needed: Possibly include a "Custom flatpak" option, such as for those that find a bug when changing some portion of the manifest (like an extension)

Reply:
I prefer GitHub Artifacts because we publicly have a reference of the manifest via MR or commit. If someone runs some random modified manifest privately then it's going to be hard for us to troubleshoot.